package.json

```diff
@@ -1,10 +1,10 @@
 {
   "version": "0.4.5",
-  "description": "Openeth Gas Stations Network",
-  "name": "@openeth/gsn",
+  "description": "Opegsn Gas Stations Network",
```

Comment:
Are we 'Opengsn' or 'OpenGSN'? Does it event have any sense to be 'Open**GSN** **G**as **S**tations **N**etwork'? 😕 
Maybe we are 'Open Gas Stations Network'?

Reply:
fixed